Changelog.md

```diff
@@ -66,7 +66,7 @@ Bug Fixes:
 * Remove empty diff marker when a diff only contains console codes. (Jon Rowe, #1506)
 * Show keyword vs hash diff marker when arguments are not `==` (Jon Rowe, #1506)
 * Change check to detect frozen objects to rescue errors rather than
-  pre-empting by checking `frozen?` due to some objects mis-behaving.
+  pre-empting by checking `frozen?` due to some objects misbehaving.
```

Comment:
If you decide to keep those changes, please add them to blame ignore https://github.com/psf/requests/blob/main/.git-blame-ignore-revs

Reply:
Dropped all changes to Changelog.md.